<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QMainWindow" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Photo Journalist Admin Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
        <item>
         <widget class="QLineEdit" name="SearchLineEdit">
          <property name="maximumSize">
           <size>
            <width>550</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>üîç –ü–æ—à—É–∫</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FilterComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>–§—ñ–ª—å—Ç—Ä–∏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="SortingComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>–°–æ—Ä—Ç—É–≤–∞–Ω–Ω—è</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="UsersTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>2</width>
              <height>2</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>120</width>
              <height>120</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>15</number>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListView" name="ObjectsList">
            <property name="sizeIncrement">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="AddButton">
          <property name="text">
           <string>–î–æ–¥–∞—Ç–∏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteButton">
          <property name="text">
           <string>–í–∏–¥–∞–ª–∏—Ç–∏</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="EditMenu">
    <property name="title">
     <string>–†–µ–¥–∞–≥—É–≤–∞–Ω–Ω—è</string>
    </property>
    <addaction name="SaveAction"/>
    <addaction name="OptionsAction"/>
   </widget>
   <widget class="QMenu" name="ProfileMenu">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>–ü—Ä–æ—Ñ—ñ–ª—å</string>
    </property>
   </widget>
   <addaction name="ProfileMenu"/>
   <addaction name="EditMenu"/>
  </widget>
  <action name="SaveAction">
   <property name="text">
    <string>–ó–±–µ—Ä–µ–≥—Ç–∏</string>
   </property>
  </action>
  <action name="OptionsAction">
   <property name="text">
    <string>–ù–∞–ª–∞—à—Ç—É–≤–∞–Ω–Ω—è</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
